packages/zosfiles/src/methods/delete/Delete.ts

```diff
@@ -193,7 +191,7 @@ export class Delete {
         ImperativeExpect.toNotBeEqual(fileSystemName, "", ZosFilesMessages.missingFileSystemName.message);
 
         // Format the endpoint to send the request to
-        const endpoint = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_ZFS_FILES + "/" + fileSystemName;
+        const endpoint = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_ZFS_FILES + "/" + encodeURIComponent(fileSystemName);
```

Comment:
curious as to why we don't use `posix.join` everywhere 😋

Reply:
Not sure, there are a ton of tests that will need to be updated in though. :)